recipes/psycopg2/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export LDFLAGS="${LDFLAGS} -L$PREFIX/lib -lssl"
+
+$PYTHON setup.py install
+
+if [[ `uname` == 'Darwin' ]]; then
+     PG_LIB=$(pg_config --libdir)
+     for LIBRARY in `find ${SP_DIR}/${PKG_NAME} -name "*.so"`;
+     do
+        install_name_tool -change libssl.1.0.0.dylib @rpath/libssl.1.0.0.dylib $LIBRARY
+        install_name_tool -change libcrypto.1.0.0.dylib @rpath/libcrypto.1.0.0.dylib $LIBRARY
```

Comment:
> It doesn't matter for conda-forge, but if someone tries to build this recipe on their own machine, this will write into the package cache.

👍 Absolutely correct. Should be a simple copy shuffle to break the hard links. Could you please raise the issue on the [feedstock]( https://github.com/conda-forge/psycopg2-feedstock ) though as this is already merged or better yet propose the change?